src/index.js

```diff
@@ -1,4 +1,29 @@
 import './style.css';
-import addDetails from '../modules/class.js';
+import displayData, { url, mainDynamicBox } from '../modules/display.js';
 
-addDetails();
\ No newline at end of file
+const nameValue = document.querySelector('.name-holder');
+const scoreValue = document.querySelector('.score-holder');
+const addBtn = document.querySelector('.submit-btn');
+const ourRefreshBtn = document.querySelector('.my-btn');
+
+addBtn.addEventListener('click', () => {
+  fetch(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      user: nameValue.value,
+      score: scoreValue.value,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((response) => response.json());
+});
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻  However According to the project requirement, the student should rewrite the code to use `await` and `async` you should Use `async` and `await` with the `fetch` method to send and receive data kindly make this

- Syntax: Following is the syntax which we could use in order to execute an asynchronous method async-await:

```
let method = async () => {
      let response = await fetch("https://.....");
      let data = await response.json();
    .....
}
```

- For more documentation you can check  [Promises vs Async Await](https://www.loginradius.com/blog/engineering/callback-vs-promises-vs-async-await/)